Simplified/ReportIssue.storyboard

```diff
@@ -77,6 +77,9 @@
                     </navigationItem>
                     <nil key="simulatedTopBarMetrics"/>
                     <nil key="simulatedBottomBarMetrics"/>
+                    <connections>
+                        <outlet property="tableView" destination="GRj-EQ-BXo" id="8Y9-oc-jsc"/>
+                    </connections>
```

Comment:
There was already a TableView in the ReportIssue.storyboard (plus the cells for Subject and Message), which was why I created an outlet for it in NYPLReportIssueViewController.swift. How would the controller file access the TableView otherwise without an outlet?